cmd/manager/main.go

```diff
@@ -101,7 +101,7 @@ func main() {
 
 	ctx := context.TODO()
 	// Become the leader before proceeding
-	err = leader.Become(ctx, "multicloudhub-operator-lock")
+	err = leader.Become(ctx, "multiclusterhub-operator-lock")
```

Comment:
Changing the name of the lock might break consensus because they won't be sharing the same lock

Reply:
reverted